src/main/java/com/plaid/client/response/AssetReportGetResponse.java

```diff
@@ -0,0 +1,220 @@
+package com.plaid.client.response;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * See https://plaid.com/docs/api/#assets for more detail into the returned schema
+ **/
+public class AssetReportGetResponse extends BaseResponse {
+  private Report report;
+  private List<Warning> warnings;
+
+  public Report getReport() { return this.report; }
```

Comment:
One last nit for you. Can we change the class to `AssetReport`?

Reply:
Yeah sorry- missed this one